src/CSS/ScrollToTop.css

```diff
@@ -0,0 +1,18 @@
+.scroll-to-top {
+    position: fixed;
+    bottom: 20px;
+    right: 20px;
+    background-color: #0D2F3F;
+    color: #fff;
+    padding: 10px 10px;
+    border-radius: 50%;
+    border: none;
+    cursor: pointer;
+    display: none; /* Initially hide the button */
+    z-index: 9999;
+  }
+  
+.visible {
+    display: block; /* Show the button when the visible class is applied */
+}
+  
```

Comment:
we are using Tailwind CSS so please use Tailwind CSS

Reply:
removed css and used Tailwind CSS